### DISCUSSION TITLE
Does LightGBMTunerCV support to use sklearn.StratifiedGroupKFold?

### DISCUSSION THREAD

**Discussion by leonleeann** [optuna/optuna#4600] [Ideas]
I need to do cross validating on a class imbalance time series to solve a binary-classification problem.
Because the samples with similar timestamp also have similar features and same target labels, the `Folding` must be done with group information. i.e. All samples from a same day should **NOT** apear in two different folds.
And because the class distribution are unbalanced, the folding must be done with `stratified`.
So, I think that I have to use `sklearn.model_selection.StratifiedGroupKFold` instead of `StratifiedKFold` and `KFold`.
But not like `KFold`( in [example](https://github.com/optuna/optuna-examples/blob/main/lightgbm/lightgbm_tuner_cv.py) ), the `split` method of `StratifiedGroupKFold` needs three arguments instead of one or two, and the last arg(groups information) is necessary!
After reading the [doc](https://optuna.readthedocs.io/en/stable/reference/generated/optuna.integration.lightgbm.LightGBMTunerCV.html) and the [example](https://github.com/optuna/optuna-examples/blob/main/lightgbm/lightgbm_tuner_cv.py) of LightGBMTunerCV, I still have **NOT** found out how to offer the groups info to the tuner.

My question is:
1. Does LightGBMTunerCV support StratifiedGroupKFold?
2. Is there a way, I can wrap `StratifiedGroupKFold` in a lambda, and then pass it to the tuner as a common `KFold`?
